content/en/docs/ops/configuration/traffic-management/dns-proxy/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# @setup profile=none
+
+set -e
+set -u
+set -o pipefail
+
+# install Istio with smart dns proxy enabled
+snip_getting_started_1
+
+# deploy test application
+snip_dns_capture_in_action_1
+
+# configure service entries and verify
+snip_dns_capture_in_action_2
+_verify_first_line snip_dns_capture_in_action_3 "$snip_dns_capture_in_action_3_out"
+snip_address_auto_allocation_1
+_verify_contains snip_address_auto_allocation_2 "*   Trying 240.240."
```

Comment:
Is checking that the IP starts with `240.240` too fragile? I wonder how reliable it will be over time?

Maybe checking that we do see a "*   Trying" line in the output, but that the IP is different now, would be better?

```suggestion
_verify_lines "
+ *   Trying
- $snip_dns_capture_in_action_3_out
"
```

Reply:
I think 240.240. prefix is correct. See https://github.com/istio/istio/blob/master/pilot/pkg/networking/core/v1alpha3/listener_test.go#L3249